WordPress/Classes/Utility/Analytics/WPAnalyticsTrackerAutomatticTracks.m

```diff
@@ -1179,6 +1179,10 @@ + (TracksEventPair *)eventPairForStat:(WPAnalyticsStat)stat
             eventName = @"post_list_button_pressed";
             eventProperties = @{ TracksEventPropertyButtonKey : @"edit" };
             break;
+        case WPAnalyticsStatPostListDuplicateAction:
+            eventName = @"post_list_button_pressed";
+            eventProperties = @{ TracksEventPropertyButtonKey : @"copy" }; // Property aligned with Android
+            break;
```

Comment:
I thought of using the new way of implementing this with using the `WPAnalyticsEvent.swift` 
 but that would make it inconsistent with the other post list button actions and the mechanism tracking the event properties (`blog_id: 174576032, button: copy, filter: Published, has_gutenberg_blocks: 1, post_id: 968, site_type: blog, type: post`) would have to be duplicated. Wdyt?

Reply:
This sounds reasonable to me, for the sake of consistency, and that it would be seemingly straightforward to migrate these actions together at once if desired in a separate refactor PR.

Reply:
Thank you for taking the time to review this @mkevins 🙇 
Ok. I agree. This refactoring should be better handled by a separate PR.